### PR TITLE
New ROC Gemini

### PR DESCRIPTION
**Pull request by Capkirk123**
Add tech nodes, configs and ECMs for the new ROC Gemini parts.

### CODE REVIEW

File: GameData/RP-0/CrewTrainingTimes.cfg [hunk 1]
```diff
@@ -111,7 +112,7 @@ TRAININGTIMES
 		mk1-3pod = Apollo
 		ROC-ApolloCM = Apollo
 		APOLLO_CM = Apollo
-	Apollo-Mission = 150
+		Apollo-Mission = 150
```
Comment (siimav) [inline GameData/RP-0/CrewTrainingTimes.cfg:115]: Why change the formatting here?

Comment (Capkirk123) [reply GameData/RP-0/CrewTrainingTimes.cfg]: Probably an accident. Fixed

**Comment by siimav** [comment]
I see that you've partially reverted the Soyuz ECM change but didnt put the `capsulesAirlock` back this time.

**Comment by Capkirk123** [comment]
Soyuz has been restored to its original state, I'll look at its suspiciously low ECMs at another time.

**Pull request merged**